packages/fx-core/src/plugins/resource/aad/aadAppClient.ts

```diff
@@ -218,6 +229,7 @@ export class AadAppClient {
         if (retries === 0) {
           throw error;
         } else {
+          TelemetryUtils.sendEvent(stage, {[Telemetry.methodName]: fn.toString(), [Telemetry.retryTimes]: (10-retries).toString()});
```

Comment:
not use magic number. we should have a param for retrynumber

Reply:
When the request succeed without retry, the retryTimes should be `0`.
So `10-retries` should be `10-retries-1`?

Reply:
If succeed, seems will not go into catch.